app/serializers/folder_serializer.rb

```diff
@@ -18,8 +18,20 @@
 #  https://github.com/puzzle/cryptopus.
 
 class FolderSerializer < ActiveModel::Serializer
-  attributes :id, :name, :description
+  attributes :id, :name, :description, :unread_transferred_count
 
-  has_many :encryptables, serializer: EncryptableMinimalSerializer
+  has_many :encryptables, serializer: EncryptableMinimalSerializer do
+    if object.personal_inbox?
+      object.encryptables.order('created_at DESC')
+    else
+      object.encryptables.order(:name)
+    end
+  end
+
+  def unread_transferred_count
+    # rubocop:disable Metrics/LineLength
```

Comment:
rubocop is telling you that this line is to long, because it's tooooooo long :joy: 

don't just add ignore statements here for this rule. rubocop should only be ignored if really necessary